tests/test_cloudpath_serialize.py

```diff
@@ -0,0 +1,14 @@
+import pickle
+
+from cloudpathlib import CloudPath
+
+
+def test_pickle_roundtrip():
```

Comment:
If there is an existing file this would be better suited to put it within let me know and happy to move it!

I verified that this does repro the issue without the above change.

Reply:
Separate file for serializing is fine. Can you also move this test over to the new file so they are together? https://github.com/drivendataorg/cloudpathlib/blob/08b018b36f90f89003e0c9e5ebd19030b41c2433/tests/test_cloudpath_file_io.py#L457-L472